lochness/email/__init__.py

```diff
@@ -167,7 +167,7 @@ def send_out_daily_updates(Lochness, days: int = 1,
 
         # too many dicom file names -> remove
         subject_mri_gb = s3_df_selected[
-                s3_df_selected.Datatype == 'mri'].groupby('subject')
+                s3_df_selected.Datatype == 'mri'].groupby('Subject')
```

Comment:
How about you post the column names of `s3_df_selected` so we are both sure?

Reply:
see line 163 in the same file